### PR TITLE
Use partials for topic pages

### PR DESCRIPTION
**Pull request by patrickpatrickpatrick**
### What

Create partials for several sections of the new pages that were being repeated

### Why

Tidy up the code somewhat and make it more maintainable 

### Visual Differences 

None, should be the same

### CODE REVIEW

File: app/views/application/_breadcrumbs.html.erb [hunk 1]
```diff
@@ -1,4 +1,7 @@
+<% inverse ||= false %>
```
Comment (chao-xian) [inline app/views/application/_breadcrumbs.html.erb:1]: With a caveat that I'm missing a ton of context coming into this fairly cold, but I'm not clear why you've set this default, and it also doesn't seem to be relevant to the intent of the commit of creating new partials - could you split this into a separate commit with more explanation around the change?

File: app/views/browse/new_show.html.erb [hunk 2]
```diff
@@ -13,24 +13,9 @@
...
-            ],
-            collapse_on_mobile: true,
-            inverse: true,
-          } %>
-        </div>
-      </div>
-    </div>
-  </div>
+  <%= render "shared/topic_page_breadcrumbs", {
+    two_thirds: true
```
Comment (chao-xian) [inline app/views/browse/new_show.html.erb:17]: Should this be `column_two_thirds`?

File: app/views/second_level_browse_page/new_show.html.erb [hunk 3]
```diff
@@ -11,48 +11,24 @@
...
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-          <%= render "govuk_publishing_components/components/breadcrumbs", {
-            breadcrumbs: [
-              {
-                title: "Home",
-                url: "/"
-              },
-              {
-                title: page.active_top_level_browse_page.title,
```
Comment (chao-xian) [inline app/views/second_level_browse_page/new_show.html.erb:25]: I may have missed a step but I can't see how this data gets passed to the new partials now?

File: app/views/shared/_topic_page_breadcrumbs.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  column_two_thirds ||= false
+
+  column_classes = %w()
+  column_classes << column_two_thirds ? "govuk-grid-column-full" : "govuk-grid-column-two-thirds"
+%>
+
+<div class="topic-page__breadcrumb-wrapper">
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="<%= column_classes.join(' ') %>">
```
Comment (hannalaakso) [inline app/views/shared/_topic_page_breadcrumbs.html.erb:11]: This outputs `false` instead of the class name.

File: app/views/shared/_topic_page_header.html.erb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+   header_wrapper_classes << "govuk-!-margin-bottom-#{margin_bottom}" if !margin_bottom.zero?
+
+   column_classes = %w()
+   column_classes << two_thirds ? "govuk-grid-column-two-thirds" : "govuk-grid-column-full"
+%>
+
+<div class="<%= header_wrapper_classes.join(' ') %>">
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="<%= column_classes.join(' ') %>">
```
Comment (hannalaakso) [inline app/views/shared/_topic_page_header.html.erb:15]: This outputs `false` instead of the class name.

**Review by hannalaakso — COMMENTED** [review]
This is a really nice tidy up of the duplicated markup, thanks @patrickpatrickpatrick 🙌  Just noticed a small issue with the class name rendering.

File: app/views/browse/new_show.html.erb (hunk 2 above)
Comment (patrickpatrickpatrick) [reply app/views/browse/new_show.html.erb]: Yes, thanks for catching that!

File: app/views/shared/_topic_page_breadcrumbs.html.erb (hunk 4 above)
Comment (patrickpatrickpatrick) [reply app/views/shared/_topic_page_breadcrumbs.html.erb:11]: was missing some parenthesis, should be working now!

File: app/views/shared/_topic_page_header.html.erb (hunk 5 above)
Comment (patrickpatrickpatrick) [reply app/views/shared/_topic_page_header.html.erb:15]: was missing some parenthesis, should be working now!

File: app/views/second_level_browse_page/new_show.html.erb (hunk 3 above)
Comment (chao-xian) [reply app/views/second_level_browse_page/new_show.html.erb:25]: Huh so it does. @hannalaakso are you able to shed any light on that? Why we were using `page.active_top_level_browse_page.title`?

Comment (hannalaakso) [reply app/views/second_level_browse_page/new_show.html.erb:25]: I think this was on me, I was mirroring the hardcoded breadcrumbs on the new level 1 templates and assumed that we needed to explicitly pass through the links for the items. @patrickpatrickpatrick's solution is definitely more robust.
